demos/display_solo.py

```diff
@@ -1,32 +1,25 @@
 """ Basic loading and visualization for the Solo robot using gepetto viewer. """
 
-import numpy as np
-import pinocchio as se3
 import time
-import os
-
 from robot_properties_solo.config import SoloConfig
+from py_gepetto_gui_helper.gepetto_gui_scene import GepettoGuiScene
+from py_gepetto_gui_helper.robot_visual import RobotVisual
+from py_gepetto_gui_helper.frame import Frame
 
-# Load the robot urdf.
-robot = SoloConfig.buildRobotWrapper()
-
-# Setup the display (connection to gepetto viewer) and load the robot model.
-robot.initDisplay(loadModel=True)
 
-# Create a first initial position for the robot. Both legs are bent inwards.
-q = robot.model.neutralConfiguration.copy()
-q[0] = 0.2
-q[2] = 0.3
-q[7:11:2] = 0.8
-q[11::2] = -0.8
-q[8:12:2] = -1.6
-q[12::2] = 1.6
+if __name__ == "__main__":
 
-# Display the configuration in the viewer.
-robot.display(q)
+    gepetto_gui_scene = GepettoGuiScene("solo_scene", "solo_window")
+    config = SoloConfig()
+    solo_visual = RobotVisual(gepetto_gui_scene, "solo", config.urdf_path,
+                              config.meshes_path)
+    solo_visual.display(config.q0)
+    # place the world frame
+    world_frame = Frame(gepetto_gui_scene)
 
-# Example of moving the robot forward and updating the display every time.
-for i in range(10):
-  q[0] += 0.05
-  robot.display(q)
-  time.sleep(0.2)
+    # Example of moving the robot forward and updating the display every time.
+    q = config.q0.copy()
```

Comment:
nit: We should fix the API that the user cannot modify the q0 and the `q0.copy()` is not needed (otherwise the user can shoot themself into the foot).